lib/Database/ImportTransaction.cpp

```diff
@@ -100,12 +100,11 @@ IDCode ImportTransaction::Implementation::addSymbolInfo(IDCode provider, StringR
   return usrCode;
 }
 
-IDCode ImportTransaction::Implementation::addFilePath(CanonicalFilePathRef canonFilePath) {
+IDCode ImportTransaction::Implementation::addFilePath(StringRef filePath) {
```

Comment:
Would probably be better to have two methods here and some comments explaining that the string one should only be used for the out files.